hz: 10              # ループ周期 [Hz]
sleep_time: 1.5     # 待機時間（他のノードの起動を待つ）[s]
target_margin: 0.25 # 車両マージン [m]
